File: sangi_practice/sangi_practice_02/model_answers/java/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt(), b = sc.nextInt(), c = sc.nextInt();
        int median = Math.max(Math.min(a, b), Math.min(Math.max(a, b), c));
        System.out.println(median);
    }
}

File: sangi_practice/sangi_practice_02/model_answers/cpp/main.cpp
#include <iostream>
#include <algorithm>
using namespace std;

int main() {
    int a, b, c;
    cin >> a >> b >> c;
    int median = max(min(a, b), min(max(a, b), c));
    cout << median << "\n";
    return 0;
}
